doc/lua_api.txt

```diff
@@ -79,6 +80,11 @@ The game directory can contain the following files:
       e.g. `disallowed_mapgen_settings = mgv5_spflags`
       These settings are hidden for this game in the world creation
       dialog and game start menu.
+    * `author`: The author of the game. It only appears when downloaded from
+                ContentDB.
+    * `release`: Ignore this: Should only ever be set by ContentDB, as it is
+                 an internal ID used to track versions.
+    * `title`: A human-readable title to address the game.
```

Comment:
hmm, I should confirm whether this does anything

Reply:
Actually it doesn't appear at all when downloading from CDB, my bad